extensions/libraries/redcore/media/redcore/translations/com_banners/banners.xml

```diff
@@ -14,7 +14,7 @@
             <field type="state" name="state" translate="1" alwaysFallback="true">State</field>
         </table>
         <component>
-            <form>com_banners#banners#id#task#!edit</form>
+            <form>com_banners#banner#id#task#!edit</form>
```

Comment:
Yeah, we did. I changed it to this at first to get out of my initial deadlock and get started on something, but somehow I never came back change the pattern. I'll remember to change it this time.